Client/game_sa/CPathFindSA.cpp

```diff
@@ -18,59 +18,25 @@ CNodeAddress* CPathFindSA::FindNthNodeClosestToCoors(CVector* vecCoors, int iNod
     CNodeAddress  node;
     CNodeAddress* nodeptr;
 
-    _asm
-    {
-        lea     eax, node
-        push    eax
-        push    0
-        push    iType  // Type : 0 - cars, 1 - boats
-        push    iNodeNumber // Nth
-        push    0
-        push    0
-        push    fDistance
-        push    0
-        mov     eax, vecCoors
-        push    [eax+8]
-        push    [eax+4]
-        push    [eax]
-        lea     eax, nodeptr
-        push    eax
-        mov     ecx, CLASS_CPathFind
-        call    dwFunc
-    }
+    // CPathFind::FindNthNodeClosestToCoors
+    ((CNodeAddress * (__thiscall*)(int, CNodeAddress*, CVector, unsigned char, float, bool, bool, int, bool, bool, CNodeAddress*))
+         FUNC_FindNthNodeClosestToCoors)(CLASS_CPathFind, nodeptr, *vecCoors, 0, fDistance, false, false, iNodeNumber, iType, false, &node);
```

Comment:
`lea eax, nodeptr`
`push eax`
```suggestion
         FUNC_FindNthNodeClosestToCoors)(CLASS_CPathFind, &nodeptr, *vecCoors, 0, fDistance, false, false, iNodeNumber, iType, false, &node);
```